src/msw/listctrl.cpp

```diff
@@ -875,6 +879,25 @@ bool wxListCtrl::GetItem(wxListItem& info) const
     return success;
 }
 
+// Check if the item is visible
+bool wxListCtrl::IsVisible(long item) const
+{
+    bool result = ::SendMessage( GetHwnd(), LVM_ISITEMVISIBLE, (WPARAM) item, 0 );
+    if( result )
+    {
+        HWND hwndHeader = (HWND) ::SendMessage( GetHwnd(), LVM_GETHEADER, 0, 0 );
+        wxRect itemRect;
+        RECT headerRect;
+		GetItemRect( item, itemRect );
+        if( Header_GetItemRect( hwndHeader, 0, &headerRect ) )
+        {
+            wxRect wxRectHeader = wxRectFromRECT( headerRect );
```

Comment:
We don't really need this variable at all, you could just use `headerRect.bottom` directly. If you do want to keep it, use the usual naming conventions.

Also, in wxMSW code, `RECT`s are usually called `rcFoo` while `wxRect`s are called `rectFoo`.

Reply:
Naming fixed.
My Windows machine with this code crashed, so I can't test it tonight.
Hopefully I will be able to do it tomorrow.